### DISCUSSION TITLE
.bcif Reader in the works?

### DISCUSSION THREAD

**Discussion by rbdavid** [MDAnalysis/mdanalysis#4654] [Development]
Are there any plans for developing a .bcif format reader so that structures downloaded from RCSB can be analyzed using MDA? I ask since there's a MMTF Reader but RCSB no longer supports that format ((https://www.rcsb.org/news/feature/65a1af31c76ca3abcc925d0c)).  

Hopefully this is not redundant with past issues/discussions. A rudimentary search didn't prove fruitful. And if there's a quick work-around, like using parmed to read the file and then port the structure into MDA, I would love to hear about it!

Thanks!

**Comment by hmacdope** [comment]
CC @richardjgowers ?

**Reply by richardjgowers** [reply]
I've not looked at this yet... It looks like rcsb provide some bindings, https://github.com/rcsb/py-mmcif https://pypi.org/project/mmcif/ so it might be possible without too much effort
